### ISSUE TITLE
Disagree with Rails Security 7.1, bullet 1

### ISSUE THREAD

**Issue by jeffdeville** [rails/rails#20012]
before_actions add functionality to a controller method.  They're usually something like authenticate_user or authorize_user

So I think that except is actually whitelisting in this instance. You want security on by default, and to have to turn it off explicitly.

But the bullet point reads:

* Use before_action only: [...] instead of except: [...]. This way you don't forget to turn it off for newly added actions.

I would suggest rewording this to:

* Use before_action exception: [...] instead of only: [...] for security-related actions. That way you don't forget to enable security checks for newly added actions.

**Comment by senny** [comment]
@jeffdeville :+1: I agree. Do you mind sending a PR?

**Comment by eileencodes** [comment]
Fixed by #20086
